envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotsVersions.kt

```diff
@@ -3,7 +3,7 @@ package pl.allegro.tech.servicemesh.envoycontrol.snapshot
 import io.envoyproxy.envoy.api.v2.Cluster
 import io.envoyproxy.envoy.api.v2.ClusterLoadAssignment
 import pl.allegro.tech.servicemesh.envoycontrol.groups.Group
-import java.util.UUID
+import java.util.*
```

Comment:
I think that some people on the team were against star imports and there is a rule in ktlint about that: https://circleci.com/gh/allegro/envoy-control/425?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

You can configure your intellij to not use star imports:
![image](https://user-images.githubusercontent.com/2753650/71238991-bebbaa80-2305-11ea-8d74-ba882e14c8b3.png)

Just check "when at least" and set it to 9999

Reply:
Done